releases/_posts/2019-04-01-spark-release-2-4-1.md

```diff
@@ -13,6 +13,8 @@ meta:
 
 Spark 2.4.1 is a maintenance release containing stability fixes. This release is based on the branch-2.4 maintenance branch of Spark. We strongly recommend all 2.4.0 users to upgrade to this stable release.
 
+In Apache Spark 2.4.1, Scala 2.12 support is GA, and it's no longer experimental. We will drop Scala 2.11 support in Spark 3.0, so please provide us feedback.
+
```

Comment:
@srowen I added this, fyi.

Reply:
BTW, should we use Scala 2.12 as default Scala version in next maintenance release before Spark 3.0?

Reply:
Looks good. If you mean, don't build 2.11, no I don't think we can do that. But otherwise it can be a default in docs, etc, in 2.4.2